custom_code/hooks.py

```diff
@@ -157,6 +161,36 @@ def target_post_save(target, created, tns_time_limit:int=5):
                     tn = TargetName.objects.create(target=target, name=alias)
                     messages.append(f'Added alias {tn.name} from TNS')
 
+            tnsphot, time_to_wait = query_TNSphot(target.name[2:],  # remove prefix
+                                    settings.BROKERS['TNS']['bot_id'],
+                                    settings.BROKERS['TNS']['bot_name'],
+                                    settings.BROKERS['TNS']['api_key'],
+                                    timelimit=tns_time_limit)
+            if tnsphot is None or (isinstance(tnsphot, Response) and tnsphot.status_code != 200):
+                if isinstance(tnsphot, Response):
+                    tns_query_status = f"TNS Request responded with code {tnsphot.status_code}!\n{tnsphot}"
+                else:
+                    tns_query_status = f'We ran out of API calls to the TNS with {time_to_wait}s left! This exceeded the {tns_time_limit}s limit!'
+                    
+                logger.info(tns_query_status)
+
+            else:
+                for candidate in tnsphot:
+                    jd = Time(candidate['jd'], format='jd', scale='utc')
+                    value = {'filter': candidate['F']}
+                    if candidate['mag']:  # detection
+                        value['magnitude'] = candidate['mag']
+                    else:
+                        value['limit'] = candidate['limflux']
+                    if candidate['magerr']:  # not empty or zero
+                        value['error'] = candidate['magerr']
+                    rd, _ = ReducedDatum.objects.get_or_create(
+                        timestamp=jd.to_datetime(timezone=TimezoneInfo()),
+                        value=value,
+                        source_name=candidate['tel']+' (TNS)',
+                        data_type='photometry',
+                        target=target)
+                
```

Comment:
This might be a remnant of a previous version when we were only getting photometry during the vetting. I recently updated it so that we also get internal names, updated coordinates, etc. (lines 95-137). Should this be deleted or are there any changes?

Reply:
Yes, you're right, this is redundant. Sorry, I didn't realize that you had made those changes on main! I deleted these lines though.